test/integration/build/outputs.tf

```diff
@@ -6,6 +6,10 @@ output "aws_vpc_id" {
   value = aws_vpc.inspec_vpc.0.id
 }
 
+output "aws_vpc_dhcp_options_id" {
```

Comment:
I see that this is currently conflicting with the output with the same name on line 25. Perhaps this one could be renamed to `aws_vpc_dhcp_opts_id` or `aws_vpc_dhcp_inspec_dopt_options_id`